packages/Webkul/Velocity/src/Resources/views/shop/checkout/cart/index.blade.php

```diff
@@ -54,12 +54,20 @@
                                 <div class="cart-item-list">
                                     @csrf
 
+                                    @php
+                                        $productOtherThanDownloadable = false;
+                                    @endphp
+
                                     @foreach ($cart->items as $key => $item)
                                         @php
                                             $productBaseImage = $item->product->getTypeInstance()->getBaseImage($item);
-
+                                            
                                             $product = $item->product;
 
+                                            if ($item->product->type != "downloadable") {
```

Comment:
I think this will be checked with the showQuantityBox method. Then on the basis of that flag, we can enable disable the button.

Reason: For downloadable, bundle and grouped quantity boxes will not be visible.